Makefile

```diff
@@ -82,6 +82,7 @@ deploy: manifests
 	kubectl wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
 	sleep 2
 	${KUSTOMIZE} build config/runtimes | kubectl apply -f -
+	${KUSTOMIZE} build config/storagecontainers | kubectl apply -f -
```

Comment:
This should be part of the default overlay.

Reply:
Does this mean add `config/storagecontainers` as part of  `config/default` resources?
I have tried that in the first version of this pr, but it didn't pass the test. due to this [issue](https://github.com/kubernetes-sigs/kustomize/issues/552). The namespace  configuration in `config/default` conflicts with `clusterstoragecontainers`. So I have to remove it from  `config/default`, and use it just like `config/runtimes`.

This also means we need to do things like this in `hack/generate-install.sh`:
the second sentence use append
```
kustomize build config/default | sed s/:latest/:$TAG/ > $INSTALL_PATH
kustomize build config/storagecontainers | sed s/:latest/:$TAG/ >> $INSTALL_PATH
```

I'm also considering another solution, to rename `config/runtimes` to `config/clusterresources`, and then we can add `clusterstorageruntimes` into `config/clusterresources`, but it may lead to more change, and I'm not sure if this is a good solution. Do you have any suggestion?

Reply:
I see, seems like we do need to move to the cluster resources. Ideally we want to minimize the number of installation files needed, maybe we can merge the runtime and storage yamls in generate-install.sh.

Reply:
updated, require approval to test